PowerEditor/installer/nativeLang/venetian.xml

```diff
@@ -975,25 +1048,25 @@ Te poi definire vari deƚimitatori de coƚona uxando spasi par spartire i divers
                 <Item id="7004" name="Órdena schede"/>
             </Window>
             <ColumnEditor title="Editor a coƚone">
-                <Item id="2023" name="Testo da inserire"/>
-                <Item id="2033" name="Nùmaro da inserire"/>
-                <Item id="2030" name="Nùmaro inisiaƚe: "/>
-                <Item id="2031" name="Incrementa de: "/>
-                <Item id="2035" name="Xeri inisiaƚi"/>
-                <Item id="2036" name="Ripeti: "/>
-                <Item id="2032" name="Formato"/>
-                <Item id="2024" name="Desimaƚe"/>
-                <Item id="2025" name="Otaƚe"/>
-                <Item id="2026" name="Exadesimaƚe"/>
-                <Item id="2027" name="Binario"/>
+                <Item id="2023" name="&amp;Testo da inserire"/>
+                <Item id="2033" name="&amp;Nùmaro da inserire"/>
+                <Item id="2030" name="Nùmaro &amp;inisiaƚe: "/>
+                <Item id="2031" name="Increment&amp;a de: "/>
+                <Item id="2035" name="&amp;Xeri inisiaƚi"/>
+                <Item id="2036" name="&amp;Ripeti: "/>
+                <Item id="2032" name="&amp;Formato"/>
```

Comment:
Sorry I just fixed an issue regarding it: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
`&amp;` in front of 'F' should be removed.

Reply:
done